test-infra/image-builder/roles/kubevirt-images/defaults/main.yml

```diff
@@ -36,6 +36,12 @@ images:
     checksum: sha256:01d9345ba7a6523d214d2eaabe07fe7b4b69b28e63d7a6b322521e99e5768719
     converted: true
 
+  debian-10:
+    filename: debian-9-openstack-amd64.qcow2
```

Comment:
debian-9-openstack-amd64.qcow2 -> **debian-10-openstack-amd64.qcow2** ?

Reply:
Good catch, fixing this in #5540